src/conversions/num_bigint.rs

```diff
@@ -111,12 +111,12 @@ macro_rules! bigint_conversion {
                     let num: Py<PyLong> =
                         Py::from_owned_ptr_or_err(py, ffi::PyNumber_Index(ob.as_ptr()))?;
                     let n_bits = ffi::_PyLong_NumBits(num.as_ptr());
-                    let n_bytes = if n_bits == -1 {
+                    let n_bytes = if n_bits == usize::MAX {
```

Comment:
Maybe the following is clearer, so that it aligns with the documentation for `_PyLong_NumBits` returning `-1` on overflow?

```suggestion
                    let n_bytes = if n_bits == (-1isize as usize) {
```